src/formatDate.js

```diff
@@ -51,6 +51,60 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const formattedDate = [];
+
+  const fromSeparator = fromFormat[3];
+  const toSeparator = toFormat[3];
+
+  const splittedDate = date.split(fromSeparator);
+
+  const fromIndexFinder = (formatToFind) => {
```

Comment:
You don't need to create this function on every `formatDate` function call.
Let's move it outside the function